spec/index.bs

```diff
@@ -5239,36 +5281,95 @@ enum GPUStoreOp {
 
     : <dfn>drawIndirect(indirectBuffer, indirectOffset)</dfn>
     ::
+        Draws primitives using parameters read from a {{GPUBuffer}}.
+
+        The <dfn dfn>indirect draw parameters</dfn> encoded in the buffer must be a tightly
+        packed block of **four 32-bit unsigned integer values (16 bytes total)**, given in the same
+        order as the arguments for {{GPURenderEncoderBase/draw()}}. Written as a C struct,
```

Comment:
I wonder how appropriate it is to refer to C structs when describing a JS API?

Reply:
Yeah, I felt weird about that too. Gave it another pass, formulating it as a chunk of JS code. Curious if that feels better. (As a piece of JS I feel an urge to turn it into a full fledged "create a buffer and upload this block to it" but that would obscure the intent to demonstrating the parameter layout.)